ui/app/hooks/tests/useCurrencyDisplay.test.js

```diff
@@ -0,0 +1,116 @@
+import assert from 'assert'
+import { renderHook } from '@testing-library/react-hooks'
+import * as reactRedux from 'react-redux'
+import { useCurrencyDisplay } from '../useCurrencyDisplay'
+import sinon from 'sinon'
+
+const tests = [
+  {
+    input: {
+      value: '0x2386f26fc10000',
+      numberOfDecimals: 2,
+      currency: 'usd',
+    },
+    result: {
+      value: '$2.80',
+      suffix: 'USD',
+      displayValue: '$2.80 USD',
+    },
+  },
+  {
+    input: {
+      value: '0x2386f26fc10000',
+      currency: 'usd',
+    },
+    result: {
+      value: '$2.80',
+      suffix: 'USD',
+      displayValue: '$2.80 USD',
+    },
+  },
+  {
+    input: {
+      value: '0x1193461d01595930',
+      currency: 'ETH',
+      numberOfDecimals: 3,
+    },
+    result: {
+      value: '1.266',
+      suffix: 'ETH',
+      displayValue: '1.266 ETH',
+    },
+  },
+  {
+    input: {
+      value: '0x1193461d01595930',
+      currency: 'ETH',
+      numberOfDecimals: 3,
+      hideLabel: true,
+    },
+    result: {
+      value: '1.266',
+      suffix: undefined,
+      displayValue: '1.266',
+    },
+  },
+  {
+    input: {
+      value: '0x3b9aca00',
+      currency: 'ETH',
+      denomination: 'GWEI',
+      hideLabel: true,
+    },
+    result: {
+      value: '1',
+      suffix: undefined,
+      displayValue: '1',
+    },
+  },
+  {
+    input: {
+      value: '0x3b9aca00',
+      currency: 'ETH',
+      denomination: 'WEI',
+      hideLabel: true,
+    },
+    result: {
+      value: '1000000000',
+      suffix: undefined,
+      displayValue: '1000000000',
+    },
+  },
+  {
+    input: {
+      value: '0x3b9aca00',
+      currency: 'ETH',
+      numberOfDecimals: 100,
+      hideLabel: true,
+    },
+    result: {
+      value: '0.000000001',
+      suffix: undefined,
+      displayValue: '0.000000001',
+    },
+  },
+]
+
+
+describe('useCurrencyDisplay', function () {
+  it('Should renturn appropriate values', function () {
+    tests.forEach(({ input: { value, ...restProps }, result }) => {
```

Comment:
We'd get a nicer test output if the `it` block was inside this loop, e.g. as demonstrated here: https://mochajs.org/#dynamically-generating-tests
Then these test cases could have more meaningful names as well.

Reply:
Oh, I see, these are taken from the old tests I'm guessing? Well I certainly won't block on them being improved anyway.

Reply:
That creates much nicer output for sure :) 

![image](https://user-images.githubusercontent.com/4448075/81716448-0373e980-943f-11ea-8f6d-ab6cb12c1279.png)

![image](https://user-images.githubusercontent.com/4448075/81716477-0a026100-943f-11ea-9119-f8e4bc11c8f0.png)

![image](https://user-images.githubusercontent.com/4448075/81716519-15558c80-943f-11ea-8742-ca97ccd4d4d9.png)